plugins/nitrokey/fu-self-test.c

```diff
@@ -10,6 +10,29 @@
 
 #include "fu-nitrokey-common.h"
 
+static void
+fu_nitrokey_version_test(void)
+{
+    /* Use the Nitrokey Storage v0.53 status response for test
+    CRC 0xa2762d14 */
+    const guint8 buf[] =  {
+    0x00,0x00,0x2e,0xef,0xc4,0x9b,0x84,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00
+    0x00,0x00,0x00,0x00,0x00,0x0c,0x2e,0x01,0x00,0x00,0x00,0x03,0x00,0x1c,0x18,0x33,
+    0x00,0x00,0x00,0x35,0x00,0x00,0x00,0x00,0x00,0x01,0x45,0x24,0xf1,0x4c,0x01,0x00,
+    0x03,0x03,0xc7,0x37,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x14,0x2d,0x76,
+    0xa2
+    };
+
+    _dump_to_console ("payload", buf, sizeof(buf));
+    NitrokeyGetDeviceStatusPayload payload;
+    memcpy (&payload, buf, sizeof(buf));
+    g_assert_cmpint (payload.VersionMajor, ==, 0);
+    g_assert_cmpint (payload.VersionMinor, !=, 0);
+    g_assert_cmpint (payload.VersionMinor, ==, 53);
```

Comment:
I think you only need to have one check on `VersionMinor`  If it matches 53 you know it's not 0.

Reply:
I wanted to make sure about this ;-) You are right of course, the `!= 0` condition will be removed.

Reply:
Cool, if you push the branch again it looks good to me with these two fixes. Thanks for tracking it down!